bakery/src/pipelines/cops.js

```diff
@@ -223,6 +224,12 @@ const pipeline = (env) => {
       taskBakeBookMetadataGroup({ image: imageOverrides }),
       taskLinkSingle({ image: imageOverrides }),
       taskMathifySingle({ image: imageOverrides }),
+      taskLinkRex({
+        image: imageOverrides,
+        inputSource: 'mathified-single',
+        inputPath: '/*.mathified.xhtml',
```

Comment:
no, it would not, this is a string that get's concatenated, but now that you have brought it to my attention, it looks like I overlooked the inconsistency compared to `taskValidateHtml` - Will change it to have it more like `taskValidateHtml`